node/service/src/parachains_db/upgrade.rs

```diff
@@ -278,6 +296,41 @@ fn paritydb_migrate_from_version_1_to_2(path: &Path) -> Result<(), Error> {
 	Ok(())
 }
 
+/// Migration from version 2 to version 3.
+/// Clears the approval voting db column which changed format and cannot be migrated.
+fn paritydb_migrate_from_version_2_to_3(path: &Path) -> Result<(), Error> {
+	parity_db::clear_column(
+		path,
+		super::columns::v2::COL_APPROVAL_DATA.try_into().expect("Invalid column ID"),
+	)
+	.map_err(|e| other_io_error(format!("Error clearing column {:?}", e)))?;
+
+	Ok(())
+}
+
+/// Migration from version 2 to version 3.
+/// Clears the approval voting db column because `OurAssignment` changed format. Not all
```

Comment:
I find this a bit worrying - what happens if enough nodes update simultaneously? Can they always recover the state they need to repopulate the database from the chain?

Reply:
AFAIK, these are not put on-chain.

Reply:
I see what you mean, makes sense